pages/_tutorials/dde/howto_create_new_profile.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How to create a new draft profile
+nextTutorial:
+  link: ./dde/new_profile
+  title: Create a New Profile in the DDE
+previousTutorial:
+  link: ./dde/update_profile
+  title: Profile Update
+
+redirect_from:
+- "/tutorials/howto/howto_create_new_profile"
+
+bioschemas:
+  "@context": https://schema.org/
+  "@type": LearningResource
+  "http://purl.org/dc/terms/conformsTo":
+  - "@type": CreativeWork
+    "@id": "https://bioschemas.org/profiles/TrainingMaterial/1.0-RELEASE"
+  about:
+    - "@id": https://schema.org
+    - "@id": http://edamontology.org/topic_0089
+  audience:
+  - "@type": Audience
+    name: (Specification developer) People interested in creating a new Bioschemas profile
+  name: "How to create a new Bioschemas profile"
+  author:
+  - "@type": Person
+    name: "Alasdair Gray"
+    "@id": https://bioschemas.org/people/AlasdairGray
+    url: https://bioschemas.org/people/AlasdairGray
+  contributor:
+  - "@type": Person
+    name: "Ricardo Arcila"
+    "@id": https://bioschemas.org/people/RicardoArcila
+    url: https://bioschemas.org/people/RicardoArcila
+  - "@type": Person
+    name: "Kenneth McLeod"
+    "@id": https://bioschemas.org/people/KennethMcLeod
+    url: https://bioschemas.org/people/KennethMcLeod
+  dateModified: 2022-11-29
+  description: "In this how-to, we will guide you through the necessary steps to create a new Bioschemas profile. The next tutorial will tell you how to add it to the DDE."
+  keywords: "schema.org, markup, structured data, bioschemas profile"
+  license: CC-BY 4.0
+  version: 2.0
+---
+
+## 1. Before you start
+
+Profiles are developed within [working groups](/groups). Before starting on a new profile, the Working Group Leads should create a new issue on the [GitHub issue tracker](https://github.com/BioSchemas/specifications/issues) to discuss with the Steering Council the need for the new profile. This will require providing use cases that the profile is aimed to satisfy. For full details, please see the [Bioschemas Governance document](https://github.com/Bioschemas/governance/blob/master/governance.md#proposing-a-new-profile).
+
+### 1.1 Use Case Development
+
+The use cases should be captured in a Markdown file that can be added to the [use case](/useCases) collection. Bioschemas use cases tend to focus on search scenarios, but some also support lightweight data exchange.
```

Comment:
Please double check that this link will work on the actual website. When I run the fork locally, it does work but it does not when I try https://bioschemas.org/useCases/

Reply:
Very basic use case page added in [224e204](https://github.com/BioSchemas/bioschemas.github.io/pull/713/commits/224e204fb21075918560ccaa42a0c59e6278c5b0)